src/_objectProps.ts

```diff
@@ -0,0 +1,42 @@
+import { Simplify } from './type-fest/simplify';
+
+export type AnyValidObjectKey = string | number | symbol;
```

Comment:
**nit:** is this not `PropertyKey`?

Reply:
TIL

I've always made a custom type for this xD